src/app/models/__tests__/form.server.model.spec.ts

```diff
@@ -157,6 +158,53 @@ describe('Form Model', () => {
         expect(actualSavedObject).toEqual(expectedObject)
       })
 
+      it('should create new date form field with default empty array of invalid days of the week and save successfully', async () => {
```

Comment:
should we extract it to its own test file? can look at https://github.com/opengovsg/FormSG/blob/2854ed2ea6147d7190aac3da11ad3696e0aaf0c5/src/app/models/field/__tests__/emailField.spec.ts#L24 for example

Reply:
Updated in latest commit